st2actions/st2actions/scheduler/config.py

```diff
@@ -50,6 +50,9 @@ def _register_service_opts():
             default='/etc/st2/logging.scheduler.conf',
             help='Location of the logging configuration file.'
         ),
+        cfg.FloatOpt(
+            'execution_scheduling_timeout_threshold_min', default=5,
```

Comment:
Can you default this to 1 min which was the original setting and behavior?  In normal use case, the scheduler shouldn't take longer than 1 minute to schedule an execution. User can adjust this up depending on their use and context.

Reply:
I will change it to 1 min.  We were seeing 2MB jsons causing this consistently, though.

Reply:
Since passing 2MB JSONs is specific to your use case, you can adjust the value in st2.conf.  But by defaulting to 5 min here, this will change the behavior for everyone else.